docs/spec/index.md

```diff
@@ -1,54 +1,67 @@
+| routes.txt | Supplement | Supplements and modifies GTFS [routes.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#routestxt) with internal route identifiers and other non-public route identification. |
+| runs_pieces.txt | TODS-Specific | Defines daily personnel schedules within a feed. |
+| run_events.txt | TODS-Specific| Defines other scheduled activities to be performed by a member of personnel during a run. |
 
-### deadhead_times.txt
+The use of the `_supplement` standard to modify other GTFS files remains experimental at this time.
+
+
+## `_supplement` Files
+
+### Structure & Evaluation
+
+The fields in all `_supplement` files match those defined in the corresponding file's [GTFS specification](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md).
+
+Each row in a `_supplement` file is used to modify the corresponding file in GTFS and should be evaluated as follows:
+
+1. If the row's Primary Key is defined in the corresponding GTFS file, and the `TODS_delete` field is equal to `1`, remove the row from the GTFS file.
+2. If the row's Primary Key is defined in the corresponding GTFS file, and the `TODS_delete` field is not defined or not equal to `1`, set or update (in the corresponding GTFS file) the value of any non-blank fields defined in the `_supplement` file's row to the `_supplement`'s value.
+3. If the row's Primary key is NOT defined in the corresponding GTFS file, add the row to the corresponding GTFS file.
+
+In other words, where a primary key matches, the row is either removed or any non-empty values in the row are used to *update* the corresponding GTFS values. Where a primary key match does not exist, the entire row is added.
+
+> _**Implications and Guidance:**_
+> - To explicitely blank a field, delete the row entirely using the `TODS_delete` field and then re-add the row with the desired values subsequent thereto.
+> - If a row contains defined values besides the Primary Key and a `TODS_delete` value of `1`, the row shall be removed and other values in that row will be ignored.
+
+### TODS-Specific Fields
+
+In addition to the fields defined in GTFS, specific fields for use within TODS are denoted by a `TODS_` field prefix.
+
+| **File** | **Field Name** | **Type** | **Required** | **Description** |
+| --- | --- | --- | --- | --- |
+| Any `_supplement` file | `TODS_delete` | Enum | Optional | (blank) - Update other fields; do not delete.<br>`1` - Deletes the GTFS row in the corresponding file whose primary key matches the value in the `_supplement` file's row. |
+| `trips_supplement.txt` | `TODS_trip_type` | Text | Optional | Defines the type of the trip if distinct from a standard revenue trip. |
+| `stops_supplement.txt` | `TODS_location_type` | Text | Optional | Defines the type of the location if distinct from a standard GTFS location type. Where defined, the GTFS `location_type` shall be ignored. |
```

Comment:
Then why not modify the `stops_supplement.txt/location_type`?

Reply:
This provides us the flexibility to implement the industry-specific location type jargon needed for our individual operations and classifications without breaking the existing GTFS data structures.

If we were to modify it directly, the values being modified in `stops.txt/location_type` would change from an enum to a string and would not be accepted by a validator. This keeps the TODS-specific datatype changes in non-GTFS fields.

Reply:
Makes sense. Is there a way to include that answer in the docs somewhere?

Reply:
In reading through things, that seems to belong more in guidance not for _use_ of the spec, but in governance of the specification. Perhaps a new eponymous "Guidance" section of governance.txt is appropriate here?